### DISCUSSION TITLE
CONFIGURAÇÃO

### DISCUSSION THREAD

**Discussion by BeAutomacaoPlax** [winder/Universal-G-Code-Sender#1834] [Help]
Olá estou fazendo um projeto e quero que ao chegar no final de ciclo ele se repita, tem algum codigo ou forma que ao chegar no final do processo ele se repita novamente?

**Comment by breiler** [comment]
No there is no such feature. You could script it by running the file from the command line:
https://github.com/winder/Universal-G-Code-Sender/tree/master/ugs-cli

The command line script is available in UGS Classic edition:
https://github.com/winder/Universal-G-Code-Sender#downloads

**Reply by BeAutomacaoPlax** [reply]
Você saberia me dizer como uso o código G para pausar?

**Reply by BeAutomacaoPlax** [reply]
Eu preciso programá-lo para que um botão reinicie o programa
